docs/reference/query-dsl/semantic-query.asciidoc

```diff
@@ -40,9 +40,209 @@ The `semantic_text` field to perform the query on.
 (Required, string)
 The query text to be searched for on the field.
 
+`chunks`::
```

Comment:
Naming nit: why not something more specific like passages? (I know we use `chunks` internally, not sure if that was a good idea now that we need to refer to it as such) 🤔